README.md

```diff
@@ -229,6 +248,57 @@ Blog.get_article_by_id(
 )
 ```
 
+## Extending Query Functions
+In the event that you want to extend QueryBuilder's functionality to include custom app specific query functions, there's the `QueryBuilder.Extension` macro to facilitate that.  You can create an extension module and use the `QueryBuilder.Extension` macro to inject all `QueryBuilder` functions into your custom module.  Any custom query functions added to your custom module are also available utilizing `QueryBuilder.from_list/2`.
```

Comment:
"the `QueryBuilder.Extension` macro" shouldn't that be "the `QueryBuilder.Extension` module"?

Reply:
I'll change to module.  The `use` is the macro.